### ISSUE TITLE
Can not select element on page

### ISSUE THREAD

**Issue by renandincer** [EagerIO/DrupalPlugin8#1]
Tested with http://tangy.io/d/8/s/admin/eager/install-app/AalP5veMma6s

![](https://cl.ly/gqsW/Image%202016-08-02%20at%202.46.45%20PM.png)

**Comment by adamschwartz** [comment]
Not sure if this is related, but it seems like there is an issue in which the preview itself is loading the Drupal admin UI instead of just being a preview of the actual site.

**Comment by speizerj** [comment]
Yeah I think it might be related, but I've spent quite a bit of time on that (possible separate) issue, and there doesn't seem to be a good way to remove it like there is on wordpress.

Disabling the admin interface on public-facing pages seems to require an additional drupal module, but that choice should obviously be left up to the site administrator. We could inject some CSS that would hide it, but that's still rather complicated and less than ideal.

If I can determine that this particular github issue is caused by the admin ui, we'll have to come up with a solution for the admin ui
